platformio.ini

```diff
@@ -764,35 +764,19 @@ lib_ignore        = Adafruit NeoPixel, SailfishLCD, SlowSoftI2CMaster, SoftwareS
 # RUMBA32
 #
 
-[common_rumba32]
-platform      = ${base_stm32.platform}
-extends       = base_stm32
-build_flags   = ${base_stm32.build_flags} -Os
-  -DSTM32F4xx -DARDUINO_RUMBA32_F446VE -DARDUINO_ARCH_STM32 "-DBOARD_NAME=\"RUMBA32_F446VE\""
-  -DSTM32F446xx "-DUSB_PRODUCT=\"RUMBA32_F446VE\""
+[env:rumba32]
+platform      = ${common_stm32.platform}
+extends       = common_stm32
+build_flags   = ${common_stm32.build_flags} 
```

Comment:
You can also rename `base_stm32` to `common_stm32`, move `-DUSBD_VID=0x0483` to its build flags, and get rid of the obsolete `common_stm32` entry.

Reply:
Good point. I've just update this as suggested.